### PR TITLE
Fix bug that editing an appointment would change it to activity

### LINKED ISSUE #285: [BUG] When appointment is edited the patient id disappears
bug as mentioned in the title, change it so that appointments are not editable.

### PR DESCRIPTION
**Pull request by ngsengleng**
### Summary

Resolves #285 

This was an issue that editCommand would create a new activity with edited fields without checking if activity is an appointment. It should work properly now.

### Checks

- [ ] Link PR to issue
- [ ] Add Reviewer
- [ ] Pass CI

### CODE REVIEW

**Comment by ngsengleng** [comment]
i dont get why the codecov is failing

File: src/main/java/gomedic/logic/commands/editcommand/EditActivityCommand.java [hunk 1]
```diff
@@ -80,6 +81,10 @@ private static Activity createEditedActivity(Activity activityToEdit,
         Title title = editActivityDescriptor.getTitle().orElse(activityToEdit.getTitle());
         Description description = editActivityDescriptor.getDescription().orElse(activityToEdit.getDescription());
 
+        if (activityToEdit.isAppointment()) {
+            PatientId patientId = editActivityDescriptor.getPatientId().orElse(activityToEdit.getPatientId());
```
Comment (simonjulianl) [inline src/main/java/gomedic/logic/commands/editcommand/EditActivityCommand.java:85]: Actually, there is no way to edit the patient id, you can just have `patientId = activityToEdit.getPatientId()`, and also you don't need to have the edit Activity Descriptor

**Review by simonjulianl — CHANGES_REQUESTED** [review]
Require some changes, don't need to change the descriptor

File: src/main/java/gomedic/logic/commands/editcommand/EditActivityCommand.java (hunk 1 above)
Comment (ngsengleng) [reply src/main/java/gomedic/logic/commands/editcommand/EditActivityCommand.java:85]: lol oops ye ill remove that

**Review by simonjulianl — APPROVED** [review]
LGTM

**Pull request merged**